solara/website/components/markdown.py

```diff
@@ -27,4 +27,13 @@ def MarkdownWithMetadata(content: str, unsafe_solara_execute=True):
                 solara.Meta(property=key, content=value)
             else:
                 solara.Meta(name=key, content=value)
-    solara.Markdown(content, unsafe_solara_execute=unsafe_solara_execute)
+    with solara.v.Html(
+        tag="div", style_="display: flex; flex-direction: row; gap: 15px; max-width: 90%; margin: 0 auto;", attributes={"id": "markdown-to-navigate"}
+    ):
+        solara.Markdown(content, unsafe_solara_execute=unsafe_solara_execute, style="flex-grow: 1; max-width: 100%;")
```

Comment:
We might want to port the `max-width: 100%;` back to the Solara component, it should remedy the mobile overflow issues Solara apps tend to have when using markdown.